docs/services/aimtti_plp.rst

```diff
@@ -1,20 +1,72 @@
 Aim TTi PL-P Series Power Supply
 ================================
+This service connects to an Aim TTi PL-P Series Power Supply. Manufacturer info can be found under: `https://www.aimtti.com/product-category/dc-power-supplies/aim-plseries <https://www.aimtti.com/product-category/dc-power-supplies/aim-plseries>`_
 
+Note how: "The New PL-P Series is the programmable (remote control) version of the New PL Series [...]", since the above
+website lists both variants (PL and PL-P).
 
+This service requires the installation of a Communications Device Class driver on the
+host PC to provide a virtual COM port instance. It can be downloaded from the manufacturer website under "Data
+and Downloads". It is also shipped with the device on a CD. Installation instructions can be found in the `instrument manual <https://resources.aimtti.com/manuals/New_PL+PL-P_Series_Instruction_Manual-Iss18.pdf>`_ ("USB Interface" on page 24).
+
+The Python API used to control the device is `dcps <https://github.com/sgoadhouse/dcps>`_
+
+Notes:
+
+- The device automatically applies a remote interface lock when it is commanded for the first time
+(see `manual <https://resources.aimtti.com/manuals/New_PL+PL-P_Series_Instruction_Manual-Iss18.pdf>`_, page 23).
+This is also noted in the `dcps library <https://github.com/sgoadhouse/dcps/blob/afbe687236bfa6176240e26790dd26b6c395b515/dcps/AimTTiPLP.py#L85>`_.
+Even after setting the interface lock to ``LOCAL``, this will instantly be overwritten back to ``REMOTE`` when a new command is issued.
+- All remote commands are listed on page 34 of the `device manual <https://resources.aimtti.com/manuals/New_PL+PL-P_Series_Instruction_Manual-Iss18.pdf>`_.
+- Only a minimum of the commands listed in the manual are implemented. More commands can easily be added in the future as needed.
+
+The service has been successfully tested with the following device:
+
```

Comment:
Shall we precise the platform on which it was tested?

Reply:
Hmm this is a good point. We know that everything was only run on Windows for now though, so let's ponder on this independently.